bears/js/JSStandardBear.py

```diff
@@ -0,0 +1,25 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.NpmRequirement import NpmRequirement
+
+
+@linter(executable='standard',
+        output_format='regex',
+        output_regex=r'  (?P<filename>[^:]+):(?P<line>\d+):(?P<column>\d+):'
+                     r' (?P<message>.+)')
+class JSStandardBear:
```

Comment:
Yes, it does.
> Automatically format code. Just run standard --fix and say goodbye to messy or inconsistent code.

See https://github.com/standard/standard#javascript-style-guide-with-linter--automatic-code-fixer